NineChronicles.Headless/GraphTypes/States/AgentStateType.cs

```diff
@@ -132,6 +133,25 @@ public AgentStateType()
                     context.Source.AgentAddress,
                     CrystalCalculator.CRYSTAL
                 ).GetQuantityString(true));
+            Field<NonNullGraphType<MeadPledgeType>>(
+                "pledge",
+                description: "mead pledge information.",
+                resolve: context =>
+                {
+                    var pledgeAddress = context.Source.AgentAddress.GetPledgeAddress();
+                    Address? address = null;
+                    bool approved = false;
+                    int mead = 0;
+                    if (context.Source.GetState(pledgeAddress) is List l)
+                    {
+                        address = l[0].ToAddress();
+                        approved = l[1].ToBoolean();
+                        mead = l[2].ToInteger();
+                    }
+
+                    return (address, contracted: approved, mead);
```

Comment:
This comment is related to the comment I left on the `MeadPledgeType.cs` file. Is it meaningful to name `approved` as `contracted`? When I saw the file, it uses `.Item2` for its `approved` field.